web/src/pages/actions/[quickActionUrlSlug].tsx

```diff
@@ -0,0 +1,89 @@
+import { Content } from "@/components/Content";
+import { HorizontalLine } from "@/components/HorizontalLine";
+import { NavBar } from "@/components/navbar/NavBar";
+import { PageSkeleton } from "@/components/PageSkeleton";
+import { TaskCTA } from "@/components/TaskCTA";
+import { TaskSidebarPageLayout } from "@/components/TaskSidebarPageLayout";
+import { getMergedConfig } from "@/contexts/configContext";
+import {
+  loadAllQuickActionUrlSlugs,
+  loadQuickActionByUrlSlug,
+  QuickActionUrlSlugParam,
+} from "@/lib/static/loadQuickActions";
+import { QuickAction, TaxFilingMethod } from "@/lib/types/types";
+import { GetStaticPathsResult, GetStaticPropsResult } from "next";
+import { NextSeo } from "next-seo";
+import { ReactElement } from "react";
+
+interface Props {
+  quickAction: QuickAction;
+}
+
+const Config = getMergedConfig();
+export const taxFilingMethodMap: Record<TaxFilingMethod, string> = {
+  online: Config.filingDefaults.onlineTaxFilingMethod,
+  "paper-or-by-mail-only": Config.filingDefaults.paperOrMailOnlyTaxFilingMethod,
+  "online-required": Config.filingDefaults.onlineRequiredTaxFilingMethod,
+  "online-or-phone": Config.filingDefaults.onlineOrPhoneTaxFilingMethod,
```

Comment:
online is not a string? but the others are?

Reply:
oops, copy paste error! this doesn't belong here at all

Reply:
But also, to answer your question because this code IS elsewhere in the app: online is a string too, here. In javascript, all keys of an object are strings and the quotation marks are optional. They're only required in the other ones because of the hyphens.

It's like how `const a = { a: 1, b: 2 }` and `const a = { "a": 1, "b": 2 }` are the exact same piece of code in JS

Reply:
Hmm, interesting and good to know about how it work underlying. Still probably don't want to mix the "" vs not "" in the same object though, just for it being pretty sake.